nusantara/nusa_datasets/nergrit/nergrit.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" NERGrit Dataset """
+
+from pathlib import Path
+from typing import List
+
+import datasets
+
+from nusantara.utils import schemas
+from nusantara.utils.common_parser import load_conll_data
+from nusantara.utils.configs import NusantaraConfig
+from nusantara.utils.constants import Tasks
+
+_CITATION = """\
+"""
+
+_LOCAL = False
+_LANGUAGES = ["ind"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_DATASETNAME = "nergrit"
+_DESCRIPTION = """\
+This NER dataset is taken from the Grit-ID repository, and the labels are spans in IOB chunking representation.
+The dataset consists of three kinds of named entity tags, PERSON (name of person), PLACE (name of location), and
+ORGANIZATION (name of organization).
+"""
+
+_HOMEPAGE = "https://github.com/grit-id/nergrit-corpus"
+_LICENSE = "MIT"
```

Comment:
Not sure if it's the right license here, if we stick to https://indonlp.github.io/nusa-catalogue/card.html?nergrit as single source of truth then it should be CC-BY-SA 4.0

Reply:
Hi @christianwbsn, thanks for the info about license. I remember again where I take the license type, I use it from https://huggingface.co/datasets/indonlu#licensing-information which is MIT.  Now, it depends on IndoNLU team which License do they prefer :-)

Reply:
I do think we should keep the original license then, which is MIT, although [the MIT license was originally designed primarily for software code (and its related documentation), while the CC licenses family was originally designed for content](https://opensource.stackexchange.com/questions/6110/is-there-any-particular-reason-to-license-docs-with-cc-by-4-0-rather-than-mit).

So for me, it's fine to keep as is then.